tests/Behat/Context/Ui/CreditMemoContext.php

```diff
@@ -142,11 +143,17 @@ public function thereShouldBeCreditMemosGenerated(int $count): void
     }
 
     /**
-     * @Then /^(\d+)(?:st|nd|rd) credit memo should be generated for order "#([^"]+)" and has total "([^"]+)"$/
+     * @Then /^(\d+)(?:st|nd|rd) credit memo should be generated for order "#([^"]+)", have total "([^"]+)" and date of being issued$/
      */
-    public function stepDefinition(int $index, string $orderNumber, string $total): void
+    public function creditMemoShouldBeGeneratedForOrderHasTotalAndDateOfBeingIssued(int $index, string $orderNumber, string $total): void
     {
-        Assert::true($this->creditMemoIndexPage->hasCreditMemoWithData($index, $orderNumber, $total));
+        $creditMemos = $this->creditMemoRepository->findBy(['orderNumber' => $orderNumber]);
+
+        uasort($creditMemos, function(CreditMemoInterface $firstCreditMemo, CreditMemoInterface $secondCreditMemo) {
```

Comment:
Instead of this one we could just pass the second argument in `findBy` method before, to sort results on repository level :)